builder/nutanix/driver.go

```diff
@@ -930,7 +930,9 @@ func (d *NutanixDriver) PowerOff(vmUUID string) error {
 			<-time.After(1 * time.Second)
 			continue
 		}
-		return fmt.Errorf("error while GetTask, %s", err.Error())
+		if err != nil {
+			return fmt.Errorf("error while GetTask, %s", err.Error())
+		}
```

Comment:
in what scenario will error not be nil here given we have a err != nil check and continue just before it?

Reply:
i think if *resp.Status == "SUCCEEDED" and there is somehow an error. seems strange but this is the line the panic is coming from.